From d1e0841a2c36362f06f3e53055eeea2f489265e9 Mon Sep 17 00:00:00 2001
From: Nicholas Eastman <29488820+ironman820@users.noreply.github.com>
Date: Fri, 14 Jul 2023 13:42:58 -0500
Subject: [PATCH] Disable 32 bit and make this pure

---
 nixGL.nix | 5 +++--
 1 file changed, 3 insertions(+), 2 deletions(-)

diff --git a/nixGL.nix b/nixGL.nix
index a4aeb55..3b7099f 100644
--- a/nixGL.nix
+++ b/nixGL.nix
@@ -12,7 +12,8 @@ nvidiaVersionFile ? null,
 # Enable 32 bits driver
 # This is one by default, you can switch it to off if you want to reduce a
 # bit the size of nixGL closure.
-enable32bits ? true
+# enable32bits ? true
+enable32bits ? false
 , writeTextFile, shellcheck, pcre, runCommand, linuxPackages
 , fetchurl, lib, runtimeShell, bumblebee, libglvnd, vulkan-validation-layers
 , mesa, libvdpau-va-gl, intel-media-driver, pkgsi686Linux, driversi686Linux
@@ -219,7 +220,7 @@ let
         runCommand "impure-nvidia-version-file" {
           # To avoid sharing the build result over time or between machine,
           # Add an impure parameter to force the rebuild on each access.
-          time = builtins.currentTime;
+          # time = builtins.currentTime;
           preferLocalBuild = true;
           allowSubstitutes = false;
         } "cp /proc/driver/nvidia/version $out 2> /dev/null || touch $out";
-- 
2.41.0

